src/main/java/org/dependencytrack/resources/v1/vo/ConciseProject.java

```diff
@@ -42,7 +42,7 @@ public record ConciseProject(
         @Schema(description = "Name of the project", requiredMode = Schema.RequiredMode.REQUIRED) String name,
         @Schema(description = "Version of the project") String version,
         @Schema(description = "Classifier of the project") Classifier classifier,
-        @Schema(description = "Whether the project is active", requiredMode = Schema.RequiredMode.REQUIRED) boolean active,
+        @Schema(description = "Timestamp of the project deactivation", type = "number", example = "1719499619599") Date inactiveSince,
```

Comment:
Similar to `Project`, the `boolean active` field should stay for API-facing classes.